metron-interface/metron-config/src/app/service/sensor-enrichment-config.service.spec.ts

```diff
@@ -174,9 +174,7 @@ describe('SensorEnrichmentConfigService', () => {
     it('deleteSensorEnrichments', () => {
       sensorEnrichmentConfigService
         .deleteSensorEnrichments('bro')
-        .subscribe(result => {
-          expect(result.status).toEqual(200);
```

Comment:
I noticed this test because of a failing Type Error. However, because we are mocking the backend and injecting the response, this test is just checking what we are injecting and is unnecessary. As a result, I removed it.